### PR TITLE
[FIX] Update tv.cpp : norm

### PR DESCRIPTION
**Pull request by LUCETE012**
norm!

### CODE REVIEW

File: Day 1/김여진/tv.cpp [hunk 1]
```diff
@@ -33,7 +33,7 @@ class TV {
```
Comment (yymin1022) [inline Day 1/김여진/tv.cpp:33]: Empty Line

File: Day 1/김여진/tv.cpp [hunk 2]
```diff
@@ -63,7 +63,7 @@ class TV {
         }
     }
 
-    void changeMode(int modeIdx)
+    void changeMode(void)
     {
         int input;
```
Comment (yymin1022) [inline Day 1/김여진/tv.cpp:68]: No New Line after Declaring New Vairable

File: Day 1/김여진/tv.cpp [hunk 3]
```diff
@@ -75,7 +75,7 @@ class TV {
         cin >> input;
         input--;
```
Comment (yymin1022) [inline Day 1/김여진/tv.cpp:77]: Empty Line

**Review by yymin1022 — CHANGES_REQUESTED** [review]

File: Day 1/김여진/tv.cpp (hunk 3 above)
Comment (LUCETE012) [reply Day 1/김여진/tv.cpp:77]: 흥

**Pull request merged**